src/lib/map.svelte

```diff
@@ -94,11 +104,37 @@
 	});
 </script>
 
+<input type="text" bind:value={searchTerm} class="search-bar" placeholder="Search" />
+<span
+	class="clear-button"
+	role="button"
+	tabindex="0"
+	on:click={clearSearch}
+	on:keydown={handleKeyDown}>x</span
+>
 <div id="map" />
 
 <style>
 	#map {
-		width: 800px;
-		height: 600px;
+		margin: 0.15rem;
+		width: 100%;
+		height: 83%;
+	}
+	.search-bar {
+		position: absolute;
+		top: 4rem;
+		left: 2.4rem;
+		z-index: 1;
+		border-radius: 0.5rem;
+		height: 1.7rem;
+		width: 14rem;
+		padding-left: 0.7rem;
+	}
+	.clear-button {
```

Comment:
I am good with the search bar having absolute positioning for now...but we should ensure the clear button has relative positioning within the search bar.

Reply:
Defintely